src/plugin/any23/ivy.xml

```diff
@@ -36,14 +36,11 @@
   </publications>
 
   <dependencies>
-    <dependency org="org.apache.any23" name="apache-any23-core" rev="2.2" conf="*->default">
+    <dependency org="org.apache.any23" name="apache-any23-core" rev="2.3" conf="*->default">
       <exclude org="org.apache.commons" name="commons-lang" />
```

Comment:
Try adding an entry for 
```
<exclude org="javax.ws.rs" name="javax.ws.rs-api" />
```

Reply:
Let me try this.

Reply:
See also [NUTCH-2669](https://issues.apache.org/jira/browse/NUTCH-2669) and [NUTCH-2697](https://issues.apache.org/jira/browse/NUTCH-2697) - short summary: the issue is fixed with ivy 2.5.0 but the upgrade would require that the local ivy cache is exclusively written by 2.5.0 and not any older ivy version, a requirement hard to ensure on shared dev or nightly build systems. Luckily, Tika dropped the javax.ws.rs-api dependency.

Reply:
After adding the exclude I was able to build it in local.